packages/wdio-types/src/Options.ts

```diff
@@ -403,6 +403,15 @@ export interface Testrunner extends Hooks, Omit<WebdriverIO, 'capabilities'>, We
     mochaOpts?: WebdriverIO.MochaOpts
     jasmineOpts?: WebdriverIO.JasmineOpts
     cucumberOpts?: WebdriverIO.CucumberOpts
+    /**
+     * autocompile options
+     */
+    autoCompileOpts?: AutoCompileConfig
+}
+
+export interface AutoCompileConfig {
```

Comment:
I like this idea of putting Babel and TS options into one property. It can be also nicely extended for other purposes in the future 👍

Reply:
I'm glad you like this idea 🎉